rust/agama-cli/src/logs.rs

```diff
@@ -304,6 +305,25 @@ fn compress_logs(tmp_dir: &TempDir, result: &String) -> io::Result<()> {
     }
 }
 
+// Sets the archive owner to root:root. Also sets the file permissions to read/write for the
+// owner only.
+fn set_archive_permissions(archive: &String) -> io::Result<()> {
+    let attr = fs::metadata(archive)?;
+    let mut permissions = attr.permissions();
+
+    // set the archive file permissions to -rw-------
+    permissions.set_mode(0o600);
+    fs::set_permissions(archive, permissions)?;
+
+    // set the archive owner to root:root
+    // note: std::os::unix::fs::chown is unstable for now
+    Command::new("chown")
```

Comment:
Just a note: it was kind of unexpected (to me) that we have an API to change the permissions but not for changing the owner.

Reply:
well, whole ```std::os::*``` seems like a wild place ;-)